### ISSUE TITLE
Iteration Plan - Feb 2021

### ISSUE THREAD

**Issue by dreamerlin** [open-mmlab/mmaction2#647]
Happy Chinese New Year!

Since MMAction2 v0.11.0 has been released, now it's time to plan for the next iteration, i.e. v0.12.0.
This will be a one-month iteration and our focus in this iteration is to support more algorithm and refactor some codes.

Algorithms:
- [x] TAM (#595)
- [x] Support TSM-MobileNetV2 (#415)

**Improvement**
- [x] Support Imgaug for augmentations in the data pipeline (#492)
- [x] Train/Test custom classes for ava dataset (#555)
- [x] Support soft label for CrossEntropyLoss (#625)

**Misc**
- [x] GPU Normalize (#586)
- [x] Support flip with label mapping (#591)
- [x] CN README (#594)
- [x] Config Refactor (#629)

delay:
~~Long-term feature banks~~
~~Add model registry~~
~~Support auto_fp16 using torch.cuda.amp~~
